darts/timeseries.py

```diff
@@ -2713,11 +2716,10 @@ def univariate_component(self, index: Union[str, int]) -> "TimeSeries":
         TimeSeries
             A new univariate TimeSeries instance.
         """
-        if isinstance(index, int):
-            new_xa = self._xa.isel(component=index).expand_dims(DIMS[1], axis=1)
-        else:
-            new_xa = self._xa.sel(component=index).expand_dims(DIMS[1], axis=1)
-        return self.__class__(new_xa)
+
+        return self.__getitem__(
+            index if isinstance(index, str) else self.components[index]
+        )
```

Comment:
in other places we use `[]` directly
```suggestion
        return self[index if isinstance(index, str) else self.components[index]]
```

Reply:
good point, thanks :)